pkg/services/ssosettings/ssosettingsimpl/service.go

```diff
@@ -250,13 +252,26 @@ func (s *Service) Delete(ctx context.Context, provider string) error {
 		return err
 	}
 
+	// When deleting settings for SAML, clear the Settings table
+	if s.configurableProviders[social.SAMLProviderName] && provider == social.SAMLProviderName {
+		samlSettings := setting.SettingsRemovals{
+			"auth.saml": make([]string, len(s.settingsProvider.Current())),
+		}
+		for k := range s.settingsProvider.Current()["auth.saml"] {
+			samlSettings["auth.saml"] = append(samlSettings["auth.saml"], k)
+		}
+		if err := s.settingsProvider.Update(setting.SettingsBag{}, samlSettings); err != nil {
+			return nil
```

Comment:
I think this should return the `err` when there was a problem with the settings removal.

Reply:
I thought about it, but what's the point of the error? Maybe we can log it, but in the future, this `settings` table will be pointless. Why block or report errors while deleting the SSO table?

Reply:
I see, then please log the error (I'd log a warn here). And make sure it does not return from here, because the `reload(..)` function will not be called if we return early.

Reply:
Fair enough. I changed the error into just a log output as requested